src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/__tests__/ArtistWrapper.test.tsx

```diff
@@ -38,6 +34,28 @@ describe("ArtistWrapper", () => {
     expect(component.state.isExpanded).toBe(!isExpanded)
   })
 
-  // it("#getRandomSVG", () => {})
-  // it("#getSVGBackground", () => {})
+  /**
+   * In CI there is a small but possible chance that this test could fail because
+   * it randomly generates the same SVG path background for wrapper and wrapper1.
+   * Unlikely, but if it does happen just re-run the same test again manually.
+   */
```

Comment:
Wouldn't the `isTest` prop you added above alleviate this risk?

Reply:
The `isTest` prop is being used to maintain a static snapshot of the entire article series. The intention here is to get have some insight into unexpected UI changes.

We still, however, want to test that the artist series is returning random SVGs as expected, so to do this I am not passing this prop here b/c we want to confirm that SVG randomization is happening as expected.